### PR TITLE
Simplify DistTime class

### PR DESCRIPTION
**Pull request by rmilfont**
Remove useless parameters, abstract the returned collection and do conversions at end phases.

Commit:Ref

TaskNumber:#23

### CODE REVIEW

File: src/DistTime.java [hunk 1]
```diff
@@ -2,27 +2,26 @@
...
 		double totalTime = Math.rint(idleTime) + packetTime;
-		double timestamp = 1;
-		ArrayList<String> vet = new ArrayList<String>();
+		double timeStamp = 1;
+		List<Integer> timeStamps = new ArrayList<>();
 		for (int j = 0; j < numberOfPackets; j++) {
-			vet.add(Integer.toHexString((int) timestamp).toUpperCase());
-			timestamp = timestamp + totalTime;
+			timeStamps.add((int) timeStamp);
+			timeStamp = timeStamp + totalTime;
```
Comment (jmarcelof) [inline src/DistTime.java]: Use the += operator here.

File: src/DistTime.java [hunk 2]
```diff
@@ -2,27 +2,26 @@
...
 		packetTime = packetSize * flitClockCycles;
 		numberOfPackets = numOfPackets;
-		ipRate = (rate / 100) * channelRate;
+		ipRate = (rate / 100);
 	}
 
-	public ArrayList<String> uniform() {
-		double idleTime = ((channelRate / ipRate - 1) * packetTime);
+	public List<Integer> uniform() {
+		double idleTime = ((1 / ipRate - 1) * packetTime);
```
Comment (jmarcelof) [inline src/DistTime.java]: I think you can put the calc of idleTime and totalTime both on constructor and make totalTime the only attribute of this class.

File: src/DistTime.java [hunk 3]
```diff
@@ -2,27 +2,26 @@
...
 		double totalTime = Math.rint(idleTime) + packetTime;
```
Comment (jmarcelof) [inline src/DistTime.java]: Put Math.rint directly on idleTime assignment.

Comment (rmilfont) [inline src/DistTime.java]: Done.

File: src/DistTime.java (hunk 1 above)
Comment (rmilfont) [inline src/DistTime.java]: Done.

File: src/DistTime.java (hunk 2 above)
Comment (rmilfont) [inline src/DistTime.java]: Done. But numberOfPackets still as atribute.

**Pull request merged**